### ISSUE TITLE
omdb private api

### ISSUE THREAD

**Issue by b-rockx** [dgilland/omdb.py#11]
omdb requires a private api key now. Please add support.

**Comment by b-rockx** [comment]
I've managed to solve the issue myself and will fork it sometime this week.

fix is to add a param to the client.py file.

`'apikey' : 'api',`

Then set the param

`title = omdb()
title.set_default('api', 'keygoeshere')
#test api key
print(title.imdbid('tt0279077'))`

**Comment by dgilland** [comment]
That approach sounds good. I'd probably just stick with just `apikey` as the name though.

**Comment by dgilland** [comment]
For others coming here for a workaround with the current version, you can do:

Using the module level omdb client:

```python
import omdb

omdb.api._client.params_map['apikey'] = 'apikey'
omdb.set_default('apikey', 'myapikey')
```

Using an separate instance of the omdb client:

```python
from omdb import Client

params_map = Client.params_map.copy()
params_map['apikey'] = 'apikey'

class OMDBClient(Client):
    params_map = params_map

# if on Python 3.5+
#class OMDBClient(Client):
#    params_map = {**Client.params_map, **{'apikey': 'apikey'}}

omdb = OMDBClient(apikey='myapikey')
```

**Comment by dgilland** [comment]
Implemented via #12
